compose-builder/.env

```diff
@@ -52,4 +52,4 @@ NATS_VERSION=2.9-alpine
 EDGEX_USER=2002
 EDGEX_GROUP=2001
 
-DEFAULT_EDGEX_RUN_CMD_PARMS="-cp=consul.http://edgex-core-consul:8500 --registry --confdir=/res"
+DEFAULT_EDGEX_RUN_CMD_PARMS="-cp=consul.http://edgex-core-consul:8500 --registry --configDir=/res"
```

Comment:
IMO we should drop the `--configDir=/res` just like was done in the Dockerfiles. Here and in all these other files unless the value will be something different than the default of  `./res` or`/res`